webpack/reducers/response_sets_reducer.js

```diff
@@ -0,0 +1,14 @@
+import {
+  FETCH_RESPONSE_SETS_FULFILLED
+} from '../actions/types';
+
+const initialState = null;
+
+export default function responseSets(state = initialState, action) {
+  switch (action.type) {
+    case FETCH_RESPONSE_SETS_FULFILLED:
+      return action.payload.data;
```

Comment:
This will create an array of response sets. We want a hash with the key being the response set id and the value being the response set. It should work with `_.keyBy(action.payload.data,'id');`